internal/stage_q1.go

```diff
@@ -44,17 +44,17 @@ func testQ1(stageHarness *test_case_harness.TestCaseHarness) error {
 		return err
 	}
 
-	L := random.RandomElementsFromArray(LARGE_WORDS, 5)
+	L := random.RandomElementsFromArray(LARGE_WORDS, 6)
 	inputs := []string{
 		fmt.Sprintf(`echo '%s %s'`, L[0], L[1]),
 		fmt.Sprintf(`echo %s     %s`, L[1], L[4]),
-		fmt.Sprintf(`echo '%s     %s' '%s''%s'`, L[2], L[3], L[4], L[0]),
+		fmt.Sprintf(`echo '%s     %s' '%s''%s' %s''%s`, L[2], L[3], L[4], L[0], L[1], L[5]),
```

Comment:
@andy1li let's check the stage instructions and see if we need to push updates there too